Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,20 @@
+
+import React from 'react';
+import './Footer.css';
+
+function Footer() {
+    let a = 1
+    let b = 3
+    let c = 5
+    let d = 7
+    return <footer>
+        <div>
+            <h4 className="comments">Comments {a}</h4>
```

Comment:
Good job using the variables in your jsx. This is exactly how you're going to want to do it when you start messing w/ state.